app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -896,7 +896,11 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
     }
 
     override fun getSystemService(name: String): Any? {
-        if (LAYOUT_INFLATER_SERVICE == name) {
+        // Issue #17759 had a crash with the PerformanceInflater.kt on Android 5.0 and 5.1
+        // when using the TimePicker. Since the inflater was created for performance monitoring
+        // purposes and that we test on new android versions, this means that any difference in
+        // inflation will be caught on those devices.
+        if (LAYOUT_INFLATER_SERVICE == name && Build.VERSION.SDK_INT > 22) {
```

Comment:
nit: can we use `Build.VERSION_CODES.LOLLIPOP_MR1` instead of `22`? It's more likely a hard-coded integer will be accidentally changed than a constant defined in the system and it's (very nitty) clearer what the intent is.